### ISSUE TITLE
SQL Exception if module disabled

### ISSUE THREAD

**Issue by mokraemer** [tine-groupware/tine#42]
The following exception is logged, if the sales module is not enabled:

ERR (3): Tinebase_Exception::logExceptionToLogger::135 Zend_Db_Statement_Exception -> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tine.tine20_sales_contracts' doesn't exist, query was: DESCRIBE `tine20_sales_contracts`
ERR (3): Tinebase_Exception::logExceptionToLogger::157 #0 .../vendor/zendframework/zendframework1/library/Zend/Db/Statement.php(303): Zend_Db_Statement_Pdo->_execute()

**Comment by mokraemer** [comment]
this is still relevant

**Comment by mokraemer** [comment]
still relevant

**Comment by pschuele** [comment]
hi @mokraemer ,
can you please give some more information?
- the complete stack trace would be great
- when is the error happening (cronjob, user json request, sync...)?

I disabled Sales in my test-environment and don't see the error.

**Comment by mokraemer** [comment]
did u call 
`php setup.php --update`

**Comment by pschuele** [comment]
yes, called --update - but I still can't reproduce it (installed 2023.11, updated to main and ran --update)....

**Comment by pschuele** [comment]
see https://github.com/tine-groupware/tine/issues/56#issuecomment-2273216792
